elements/rh-footer/rh-footer.css

```diff
@@ -16,18 +13,19 @@
   --pfe-accordion--BackgroundColor--active: transparent;
   --pfe-accordion--BackgroundColor--expanded: #151515;
   --pfe-accordion--BorderColor: var(--_border-color);
+  --pfe-accordion--FontSize--header: initial;
   --pfe-accordion--FontWeight--header: 300;
   --pfe-accordion--accent--expanded: var(--_accent-color);
   --pfe-accordion--accent--active: var(--_accent-color);
 
   /* apply sensible defaults based on redhat standards. */
   color: #fff;
   font-family: 'Red Hat Text', 'RedHatText', 'Overpass', Overpass, Arial, sans-serif;
-  line-height: 125%;
+  line-height: 150%;
   font-weight: 300;
 
   /* set at 18px for margin and padding standardization */
-  font-size: 18px;
+  font-size: initial;
```

Comment:
This is the meat and potatoes of making ems work properly. It overrides the websites base font-size and uses the user's browser settings.

Reply:
@heyMP Not sure where to put this feedback, but the column titles (`Products`, `Tools`, `Try, buy, sell`, etc.) are rendering `"Red Hat Display"` instead of `"Red Hat Text"`. Everything else looks fantastic.

Reply:
I checked this using the local dev server, and this was the result:
<img width="1392" alt="Screen Shot 2022-07-05 at 10 04 44" src="https://user-images.githubusercontent.com/1466420/177269525-8c3e752d-644a-4986-8890-14bbe972f3ff.png">

In this screenshot, the heading elements use the `"Red Hat Display"` font, as specified in the page's (light DOM) `main.css`.

I added a CSS rule which sets the font-family for slotted headings to the `--rh-font-family-heading` token, using `!important` to overrule page CSS. The result was this screenshot, where you can see the headings use the `"RedHatDisplay"` font, which, confusingly, is a different font.

<img width="1392" alt="Screen Shot 2022-07-05 at 10 11 14" src="https://user-images.githubusercontent.com/1466420/177270889-e08f1a68-a457-4541-94fa-222c8d353a19.png">

@heyMP I pushed this commit, but feel free to revert if you have a better solution. `!important` is perhaps a bit upsetting, but OTOH, users can always set `--rh-font-family-heading` - we also take this approach in button.

Reply:
Sounds good.  We've been using !important when absolutely necessary.  Makes sense to me.  Thank you~!